icon-hider@kalnitsky.org/_config.js

```diff
@@ -11,10 +11,10 @@
 
 // extension name
 // used as identifier when adding to status area
-const EXTENSION_NAME = 'Icon Hider';
+var EXTENSION_NAME = 'Icon Hider';
```

Comment:
Why this change? What's wrong with `const`?

Reply:
GNOME logs warnings when using `const`
>Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.

This change isn't strictly necessary, and I can undo it if you object. I was mostly trying to get quieter logs when debugging. Per [this stackoverflow answer](https://stackoverflow.com/a/32558929), imports cannot be reassigned by the code that imports them, so the behavior here should still be consistent.